src/Airbrake/Notifier.php

```diff
@@ -144,7 +144,7 @@ public function sendNotice($notice)
     {
         foreach ($this->filters as $filter) {
             $notice = $filter($notice);
-            if ($notice === false) {
+            if ($notice === null || $notice === false) {
```

Comment:
We can, but I prefer to be strict about data we accept (especially since PHP has very strange design regarding falsey values).